packages/edit-site/src/components/sidebar-navigation-screen-navigation-menu/index.js

```diff
@@ -74,6 +116,14 @@ export default function SidebarNavigationScreenNavigationMenu() {
 
 	return (
 		<SidebarNavigationScreenWrapper
+			actions={
+				<ScreenNavigationMoreMenu
+					menuTitle={ decodeEntities( menuTitle ) }
```

Comment:
```suggestion
					menuTitle={ menuTitle }
```

Let's pass the raw value here and allow the `ScreenNavigationMoreMenu` to handle output.